lighthouse-core/gather/drivers/extension.js

```diff
@@ -152,6 +152,22 @@ class ExtensionDriver extends Driver {
   getCurrentTabURL() {
     return this.queryCurrentTab_().then(tab => tab.url);
   }
+
+  getCurrentTabId() {
+    return new Promise((resolve, reject) => {
+      this.queryCurrentTab_().then(currentTab => {
+        chrome.debugger.getTargets(targets => {
+          const tab = targets.find(tab => tab.tabId === currentTab.id);
+
+          if (!tab) {
+            reject(new Error('We can\'t find a target id.'));
+          }
+
+          resolve(tab.id);
```

Comment:
yes .id just is an index 1, 2, 3...
but I need the targetid so i can match with it in the service worker